### PR TITLE
Fix - Update Arc Flagged Statuses

### PR DESCRIPTION
**Pull request by andrewskeaIS**
## Description

Updated regex for phone numbers
Removed caching for all views 
Ppdate arc flagged statuses to false upon successful data submission

## Todo's before PR

- [x] Branch has been rebased against develop
- [x] Unit tests passed (`make test`)
- [x] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates have been checked against the relevant user-story

## Migrations

- [x] Null fields in models specifies default value
- [x] You generated and applied migrations (`make migrate`)
- [x] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assignees (those who'll merge)
- [x] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/views/declaration.py [hunk 1]
```diff
@@ -407,3 +410,29 @@ def generate_list_of_updated_tasks(application_id):
         updated_list.append('References')
 
     return updated_list
+
+
+def clear_arc_flagged_statuses(application_id):
+    """
+    Method to
```
Comment (kingainformed) [inline application/views/declaration.py]: Can you add some method commentary here?

Comment (Thomas-95IS) [reply application/views/declaration.py]: Have done so in latest commit.

**Comment by Thomas-95IS** [comment]
Aborted 7b484b0 build to start build on new commit.

**Comment by andrewskeaIS** [comment]
run jenkins build

**Comment by andrewskeaIS** [comment]
rerun jenkins build

**Pull request closed without merging**